src/Toggle/types.ts

```diff
@@ -0,0 +1,8 @@
+import { ChangeEventHandler } from 'react';
+
+export interface ToggleProps {
```

Comment:
What do you think if we add this?
```suggestion
export interface ToggleProps extends BaseSpacingProps, React.HTMLAttributes<HTMLInputElement>{
```

Reply:
Not sure if we want to Extend Base Spacing here. Elements should have sizes by design and spacing should be given by the parent. Like: 

```jsx
<Box m={1}>
  <Toggle checked={checked} onChange={_onChange} />
</Box>
```

Reply:
We are already wrapping in the input in a div. if we do this in the DOM we will have something like:
```
<div m={1}>
  <div>
    <input />
  </div>
</div>
```

And if we add the spacing to the component the final DOM will be:
```
<div m={1}>
  <input />
</div>
```

I do not say that is wrong or not. Just that we need to know this

Reply:
Use BaseSpacing just for margin is a very good idea and it may be very useful. But what happens with padding? How is this padding supposed to look like?

Reply:
Good question. Maybe we just need to modify it so they can handle just the margin?
But padding and margin should work the same if you do not have a border no? Because in this case the padding is not going to the input, is going to the wrapper